app/code/Academy/ProductFeed/view/adminhtml/ui_component/product_feed_manage_form.xml

```diff
@@ -0,0 +1,239 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<form xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Ui:etc/ui_configuration.xsd">
+    <argument name="data" xsi:type="array">
+        <item name="js_config" xsi:type="array">
+            <item name="provider"
+                  xsi:type="string">product_feed_manage_form.product_feed_manage_form_data_source
+            </item>
+            <item name="deps"
+                  xsi:type="string">product_feed_manage_form.product_feed_manage_form_data_source
+            </item>
+        </item>
+        <item name="label" xsi:type="string" translate="true">Product Feed Information</item>
+        <item name="layout" xsi:type="array">
+            <item name="type" xsi:type="string">tabs</item>
+            <item name="navContainerName" xsi:type="string">left</item>
+        </item>
+        <item name="config" xsi:type="array">
+            <item name="dataScope" xsi:type="string">data</item>
+            <item name="namespace" xsi:type="string">course_form</item>
+        </item>
+        <item name="buttons" xsi:type="array">
+            <item name="save" xsi:type="array">
+                <item name="name" xsi:type="string">save</item>
+                <item name="label" xsi:type="string" translate="true">save</item>
+                <item name="class" xsi:type="string">primary</item>
+            </item>
+            <item name="back" xsi:type="array">
+                <item name="name" xsi:type="string">back</item>
+                <item name="label" xsi:type="string" translate="true">Back</item>
+                <item name="class" xsi:type="string">back</item>
+                <item name="url" xsi:type="string">*/*/</item>
+            </item>
+        </item>
+    </argument>
+    <dataSource name="product_feed_manage_form_data_source">
+        <argument name="dataProvider" xsi:type="configurableObject">
+            <argument name="class" xsi:type="string">Academy\ProductFeed\Model\DataProvider</argument>
+            <argument name="name" xsi:type="string">product_feed_manage_form_data_source</argument>
+            <argument name="primaryFieldName" xsi:type="string">product_feed_id</argument>
+            <argument name="requestFieldName" xsi:type="string">id</argument>
+        </argument>
+        <argument name="data" xsi:type="array">
+            <item name="js_config" xsi:type="array">
+                <item name="component" xsi:type="string">Magento_Ui/js/form/provider</item>
+                <item name="submit_url" xsi:type="url" path="*/*/save"/>
+            </item>
+        </argument>
+    </dataSource>
+    <fieldset name="general">
+        <argument name="data" xsi:type="array">
+            <item name="config" xsi:type="array">
+                <item name="collapsible" xsi:type="boolean">false</item>
```

Comment:
wydaje mi się, że to jest wartość domyślna więc jest tu niepotrzebne, chyba, że się mylę to może zostać

Reply:
Faktycznie. Skoro korzystam z tabów jest to niepotrzebne. Dzięki